multisig/proposals/sepolia/vip-035/index.ts

```diff
@@ -0,0 +1,84 @@
+import { parseUnits } from "ethers/lib/utils";
+
+import { NETWORK_ADDRESSES } from "../../../../src/networkAddresses";
+import { makeProposal } from "../../../../src/utils";
+
+const { sepolia } = NETWORK_ADDRESSES;
+
+export const UPPER_BOUND_RATIO = parseUnits("1.01", 18);
+export const LOWER_BOUND_RATIO = parseUnits("0.99", 18);
+export const ONE_JUMP_MAIN = "0xB15CA4Cd4bA5696FDC1cbBd859588c0421cc1F68";
+export const ONE_JUMP_PIVOT = "0xDAF249b4A937385dB567B73630539BacDBB04342";
```

Comment:
Nitpick. Instead of using main and pivot, we could name them `rsETH_ONE_JUMP_REDSTONE_ORACLE` and `rsETH_ONE_JUMP_CHAINLINK_ORACLE`, clearly identifying what they are. How they are used is defined in the command `setTokenConfig`

Reply:
[49b54fb](https://github.com/VenusProtocol/vips/pull/304/commits/49b54fbfc4f23900ab10e87b610f4d4acc5f1bd0)